packages/eslint-plugin/ROADMAP.md

```diff
@@ -34,7 +34,7 @@
 | [`promise-function-async`]        | ✅  | [`@typescript-eslint/promise-function-async`]        |
 | [`typedef`]                       | 🛑  | N/A                                                  |
 | [`typedef-whitespace`]            | ✅  | [`@typescript-eslint/type-annotation-spacing`]       |
-| [`unified-signatures`]            | 🛑  | N/A                                                  |
+| [`unified-signatures`]            | ✅  | [`@typescript-eslint/unified-signatures`]            |
```

Comment:
need to add the link at the bottom of the file or else this won't work

Reply:
Fixed.